### DISCUSSION TITLE
Armor Trims

### DISCUSSION THREAD

**Discussion by Shahelion** [Roadhog360/Et-Futurum-Requiem#461] [Ideas]
Probably only limited to vanilla armor, although if you found a way to make it work on other modded armor sets that would be amazing. Perhaps some separate overlay layer? Trim cosmetic Baubles?

**Comment by Roadhog360** [comment]
I was going to develop it with mixins to make it work with most armor (I'd try to port the armor stand preview thing so you can see if it works with a modded piece before applying it)

Currently I am not sure where to begin with it though and it is not high up in priorities, but this is actually something that is on my plans.
